stage0/run.go

```diff
@@ -466,5 +468,5 @@ func launchMetadataService(debug bool) error {
 		Stderr: os.Stderr,
 	}
 
-	return cmd.Start()
+	return cmd.Run()
```

Comment:
+1
> @eyakubovich At this stage dropping --spawn-metadata-service and having that as a separate step sgtm. IMHO it will make it clearer for people to understand conceptually (that it is a different service, what it provides, that it needs to be managed, etc)